plugins/ptp_operator/ptp_operator_plugin.go

```diff
@@ -175,6 +160,73 @@ func Start(wg *sync.WaitGroup, configuration *common.SCConfiguration, fn func(e
 	return nil
 }
 
+// getCurrentStatOverrideFn is called when current state is received by rest api
+func getCurrentStatOverrideFn() func(e v2.Event, d *channel.DataChan) error {
+	return func(e v2.Event, d *channel.DataChan) error {
+		if e.Source() != "" {
+			log.Infof("setting return address to %s", e.Source())
+			d.ReturnAddress = pointer.StringPtr(e.Source())
+		}
+		log.Infof("got status check call,send events for subscriber %s => %s", d.ClientID.String(), e.Source())
+		var eventType ptp.EventType
+		if !strings.Contains(e.Source(), string(ptp.PtpLockState)) &&
```

Comment:
could just add a `else` statement at line 184

Reply:
can we turn off that  earl return linter error?